homework-unit7/chat/index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>chat</title>
+</head>
+<body>
+    <div id="container">
+        <aside>
+            <header>
+                <input type="text" placeholder="search">
+            </header>
+            <ul>
+                <li>
+                    <img src="./img/vincent-porter.jpg" alt="">
+                    <div>
+                        <h2>Vincent Porter</h2>
+                        <h3>
+                            <span class="status green"></span>
+                            online
+                        </h3>
+                    </div>
+                </li>
+                <li>
+                    <img src="./img/aiden-chavez.jpg" alt="">
+                    <div>
+                        <h2>Aiden Chavez</h2>
+                        <h3>
+                            <span class="status orange"></span>
+                            left 7 mins ago
+                        </h3>
+                    </div>
+                </li>
+                <li>
+                    <img src="./img/mike-thomas.jpg" alt="">
+                    <div>
+                        <h2>Mike Thomas</h2>
+                        <h3>
+                            <span class="status green"></span>
+                            online
+                        </h3>
+                    </div>
+                </li>
+                <li>
+                    <img src="./img/erica-hughes .jpg" alt="">
+                    <div>
+                        <h2>Erica Hughes</h2>
+                        <h3>
+                            <span class="status green"></span>
+                            online
+                        </h3>
+                    </div>
+                </li>
+                <li>
+                    <img src="./img/ginger-johnston .jpg" alt="">
+                    <div>
+                        <h2>Ginger Johnson</h2>
+                        <h3>
+                            <span class="status green"></span>
+                            online
+                        </h3>
+                    </div>
+                </li>
+                <li>
+                    <img src="./img/tracy-carpenter .jpg" alt="">
```

Comment:
```suggestion
                    <img src="./img/tracy-carpenter.jpg" alt="">
```
Xóa toàn bộ ký tự đặc biệt đi
Tên file, folder, biến, hàm, ... không được phép chứa ký tự đặc biệt